### PR TITLE
Split output and logs

### PR DESCRIPTION
**Pull request by MatthewJohn**
Issue #505

### CODE REVIEW

File: lib/logging.go [hunk 1]
```diff
@@ -17,6 +19,22 @@ var (
 	TraceLogging         = slog.Levels{slog.PanicLevel, slog.FatalLevel, slog.ErrorLevel, slog.WarnLevel, slog.InfoLevel, slog.NoticeLevel, slog.DebugLevel, slog.TraceLevel}
 )
 
+func NewStderrConsoleWithLF(lf slog.LevelFormattable) *handler.ConsoleHandler {
+	h := handler.NewIOWriterWithLF(os.Stderr, lf)
+
+	// default use text formatter
+	f := slog.NewTextFormatter()
+	// default enable color on console
+	f.WithEnableColor(color.SupportColor())
```
Comment (yermulnik) [inline lib/logging.go:28]: Not sure how this works as it still produces color output for me when redirecting to a file which in general should not happen with color output.
Btw we have a feature request to enable/disable color based on condition: https://github.com/warrensbox/terraform-switcher/issues/462 =)

**Review by yermulnik — COMMENTED** [review]
Perfect. LGTM. Works for me. Though since it's sort of essential change in existing behavior, I'd summon @warrensbox to have him approve it.

File: lib/logging.go (hunk 1 above)
Comment (MatthewJohn) [reply lib/logging.go:28]: Were you hoping anything to be done in this PR or wait for #462  - as suggested, this is copied from upstream for now, to just change the output :)

Comment (yermulnik) [reply lib/logging.go:28]: Nope =) PR looks (and works) good to me. Just need Warren to review and approve I guess.

**Review by warrensbox — APPROVED** [review]

**Pull request merged**